src/app/cid/[cid]/page.tsx

```diff
@@ -0,0 +1,58 @@
+'use client'
+
+import Link from "next/link"
+import { notFound } from "next/navigation"
+import { SimpleError } from "@/components/error"
+import { Loader } from "@/components/brand"
+import { useContentCid, useShardCid } from "@/hooks/cid"
+
+export const runtime = 'edge'
+
+function Space ({ space }: { space: { did: string, insertedAt: string } }) {
+  return (
+    <div className='flex flex-row justify-between'>
+      <div className='w-7/12 overflow-hidden text-ellipsis'>
+        <Link className='underline text-blue-200' href={`/spaces/${space.did}`}>{space.did}</Link>
+      </div>
+      <div className='w-4/12'>{space.insertedAt}</div>
+    </div>
+  )
+}
+
+export default function Customer ({ params: { cid } }: { params: { cid: string } }) {
+
+  const { data: contentData, error: contentError, isLoading: contentIsLoading } = useContentCid(cid)
+  const { data: shardData, error: shardError, isLoading: shardIsLoading } = useShardCid(cid)
```

Comment:
The same CID for both?

Reply:
yep - the idea here is that the the user doesn't know what kind of a CID they have, so we look in both places - I don't think we can reliably distinguish between a shard CID and a content CID just by looking at it (ie, what if someone grabbed a shard CAR and uploaded it as a piece of content?) so we do need to look it up in both places - if there is a way to tell just by looking at the CID I'd definitely be happy to implement that here!